app/Util/BusinessTravelPolicy.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Util;
+
+class BusinessTravelPolicy
+{
+    public const TRIP_FEE_COST_RATIO = 0.2;
```

Comment:
Shouldn't this be in config? Also, why 20%?

Reply:
Also arbitrary

Reply:
The travel policy wiki page suggests 10% so PMs might be confused. imo leave it lower and let that be a management decision instead of a software-enforced decision.

Reply:
Yeah, I know. I've had conversations with Rahul and I think the policy this year will be higher than 10%, but I'll leave it configurable.